Manifests/ManifestsApple/com.apple.applicationaccess-tvOS.plist

```diff
@@ -683,8 +683,6 @@ The payload organization for a payload need not match the payload organization i
 			<integer>1</integer>
 			<key>pfm_title</key>
 			<string>Deferred Software Updates Delay</string>
-			<key>pfm_tvos_min</key>
-			<string>12.2</string>
```

Comment:
Can we leave this one and remove the one on line 677?  Want to keep it in alphabetical order.

Reply:
I got ya.